_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,69 @@
+## `content_langs`
+
+{{ site.requires_5_9v }}
+
+This option allows you to override the languages displayed in the `language` dropdown toolbar button and the `language` menu item. This option accepts an array of language objects.
+
+Each language object is a JavaScript object defined using the following properties.
+
+| Name | Type | Requirement | Description |
+| ---- | ---- | ----------- | ----------- |
+| title | String | required | The name shown for this language in the UI. |
+| code | String | required | The language tag, as defined in [Tags for Identifying Languages (BCP47)](https://www.ietf.org/rfc/bcp/bcp47.txt), for this language. This tag is placed in the `lang` attribute for elements formatted using this language. |
+| customCode | String | optional | An alternative language tag, that does not need to be compatible with BCP47. This code is placed in the `data-mce-lang` attribute for elements formatted using this language, if it is defined. |
+
+**Type:** `Array<ContentLang>`
+
+**Default Value:**
```

Comment:
This information is no longer correct as was mentioned on Wednesday. This will not have a default moving forward, so this needs changing to reflect that.